### ISSUE TITLE
Content Edit: VA 508 Office

### ISSUE THREAD

**Issue by jknipes** [department-of-veterans-affairs/va.gov-team#98614]
## Issue Description
Per [Footer ticket 88914](https://github.com/orgs/department-of-veterans-affairs/projects/1463/views/1?pane=issue&itemId=84087708&issue=department-of-veterans-affairs%7Cva.gov-team%7C88914), make content edits to the [Request support from the VA Section 508 Office](https://depo-platform-documentation.scrollhelp.site/developer-docs/request-support-from-the-va-508-office) page linking to the Support Slack channel.

Original Request:
>When we request a 508 audit we need to identify who the 508 office contact is.  For OCTO products it is Christopher Walker. Can we add this to the documentation?
This question is asked after "Product Type" 

PC Team Response:
>The PW page that you link to has the following instructions listed within it:

>"If you do not have a regular contact, ask the Product Support Team for a reference."

>As a rule, we do not list personal names on the PW. (we are in the process of auditing the entire PW for personal names) Instead, we list team names and slack channels. Would it help you if we hyperlinked Platform Support Team so that it opened in slack? This would make those instructions stand out more on the PW page and it would help the user get the info they need faster.

Author suggests:
@BSmolensky - Thanks for your comment - yes, I think it would be helpful to add

> To find out who the 508 office contact is for OCTO products, contact the `Platform Support Team` (and hyperlink to slack). Slack Channel: [vfs-platform-support](https://dsva.slack.com/archives/CBU0KDSB1).

#### In `6. Fill out the form with the relevant data for your request.` add: `As of February 2025`
---
## Tasks
- [x] Make changes according to Description
- [x] Check for launch blockers
- [x] Check for Style Guide violations (Steps on this page could be better broken up, bold type, quotations, etc.)
- [ ] If found, create separate ticket in Current Sprint for Style guide work
- [ ] if bandwidth permits, fix violations now 
- [x] if unable to fix violations now, move the new ticket to Backlog

## Acceptance Criteria
- [x] Page is updated and ready to deploy
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by moviedosoco** [comment]
@jknipes Changes were made according to Description. 
I also added the date to the page, and ending periods in “Document Review” copy. Still working on the Style Guide violations. Thank you.

**Comment by jknipes** [comment]
Moving to current sprint due to available capacity.

**Comment by moviedosoco** [comment]
@jknipes 
- No launch blockers found. 
- The copy in number 6 (Audit request) was modified according to the Description. 
- We need to open a new ticket for Style Guide violations.

**Comment by jknipes** [comment]
@moviedosoco can you please add the Style Guide violations you found to [the description in this ticket](https://github.com/orgs/department-of-veterans-affairs/projects/1463/views/1?filterQuery=va+508&pane=issue&itemId=91522013&issue=department-of-veterans-affairs%7Cva.gov-team%7C99495) (bullet points)?

**Comment by jknipes** [comment]
Closing as _Done_
